piet-gpu/shader/bbox_clear.comp

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: Apache-2.0 OR MIT OR Unlicense
+
+// Clear path bbox to prepare for atomic min/max.
+
+#version 450
+#extension GL_GOOGLE_include_directive : enable
+
+#include "mem.h"
+#include "setup.h"
+
+#define LG_WG_SIZE 9
+#define WG_SIZE (1 << LG_WG_SIZE)
+
+layout(local_size_x = WG_SIZE, local_size_y = 1) in;
```

Comment:
Some older Android phones limit workgroup sizes to 128, and the old pipeline has a setting to support that. Do you plan to have a similar facility in the new pipeline?

Reply:
Thanks for the reminder. Yes, I'll definitely make sure to make this compatible. It's mostly just an issue of more repetition in the ninja build file and the various code to instantiate the stages.